dydx/dydxViews/dydxViews/_v4/Portfolio/Components/Sections/dydxPortfolioPendingPositionsItemViewModel.swift

```diff
@@ -0,0 +1,131 @@
+        .init(marketLogoUrl: URL(string: "https://v4.testnet.dydx.exchange/currencies/eth.png"),
+              marketName: "ETH-USD",
+              margin: "$1000.00",
+              orderCount: 2,
+              viewOrdersAction: {},
+              cancelOrdersAction: {}
+        )
+    }()
+
+    private var topContent: some View {
+        VStack(spacing: 8) {
+            HStack(spacing: 8) {
+                PlatformIconViewModel(type: .url(url: marketLogoUrl),
+                                      clip: .defaultCircle,
+                                      size: CGSize(width: 20, height: 20))
+                .createView()
+                Text(marketName)
+                    .themeFont(fontSize: .small)
+                    .themeColor(foreground: .textSecondary)
+                Spacer()
+            }
+            HStack(spacing: 0) {
+                Text(localizerPathKey: "APP.GENERAL.MARGIN")
+                    .themeFont(fontSize: .smaller)
+                    .themeColor(foreground: .textTertiary)
+                Spacer()
+                Text(margin)
+                    .themeFont(fontSize: .smaller)
+                    .themeColor(foreground: .textSecondary)
+            }
+        }
+        .padding(.vertical, 10)
+    }
+
+    private var divider: some View {
+        Spacer(minLength: 1)
+            .overlay(ThemeColor.SemanticColor.borderDefault.color)
+    }
+
+    private var bottomContent: some View {
+        let viewOrdersStringKey: String
+        let viewOrdersStringParams: [String: String]?
+        if orderCount > 1 {
+            viewOrdersStringKey = "APP.GENERAL.VIEW_ORDER"
+            viewOrdersStringParams = nil
+        } else {
+            viewOrdersStringKey = "APP.GENERAL.VIEW_ORDERS_COUNT"
+            viewOrdersStringParams = ["NUM_ORDERS": "\(orderCount)"]
+        }
+
+        let viewOrders = Text(localizerPathKey: viewOrdersStringKey, params: viewOrdersStringParams)
+            .themeFont(fontSize: .smaller)
+            .themeColor(foreground: .colorPurple)
+            .padding(.vertical, 8)
+        let cancel = Text(localizerPathKey: "APP.GENERAL.CANCEL")
+            .themeFont(fontSize: .smaller)
+            .themeColor(foreground: .colorRed)
+            .padding(.vertical, 8)
+        return HStack(spacing: 0) {
+            Button(action: viewOrdersAction, label: { viewOrders })
+            Spacer()
+            Button(action: cancelOrdersAction, label: { cancel })
+        }
+    }
+
+    public override func createView(parentStyle: ThemeStyle = ThemeStyle.defaultStyle, styleKey: String? = nil) -> PlatformView {
+        PlatformView(viewModel: self, parentStyle: parentStyle, styleKey: styleKey) { [weak self] _  in
+            guard let self = self else { return AnyView(PlatformView.nilView) }
+
+            return VStack(spacing: 0) {
+                self.topContent
+                self.divider
```

Comment:
Doesn't seem like the divider is visible from the screenshot

Reply:
addressed in de39f47380e9b165ff43b1afa2156f96029a9dd4
![Screenshot 2024-06-17 at 1 56 50 PM](https://github.com/dydxprotocol/v4-native-ios/assets/149746839/b0210c30-4488-4a45-bc7c-410da9402824)